CONTRIBUTING.md

```diff
@@ -63,6 +63,11 @@ npm run dev
 
 On Visual Studio Code, open the repository. Then on the top bar, select Run, then Start Debugging (F5).
 
+```
+cd extension
+npm install
+```
```

Comment:
Can you move the commands above line 64?

Reply:
done

Reply:
Sorry, I wasn't clear enough. The commands should go above the line "On Visual Studio Code, open the repository. Then on the top bar, select Run, then Start Debugging (F5)."